### ISSUE TITLE
PP6 People Profile Custom Listing needs to update page title when filtered by category

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#657]
When filtered by category, the PP6 custom listing view pane does not update the visible title of the page automatically.

For example, if you have taxonomy term "Featured People" with a term ID of 1 and you visit https://www.uoguelph.ca/site/people/1, the title in the <title> element will be "Featured People" but the visible title on the page will be "Our People".

Fix likely involves:
1. Go to PP6 view
2. Under Pane Settings, edit the Argument Input
3. Set "Content: Has taxonomy term ID source" to "From panel argument"
4. Set Panel argument to "First"

Note that if you switch the /people/!tag page so that it uses Custom Listing, you'll need to enable the Panel Title on the Panel / Variant / Content page.

![pp6-page-update](https://cloud.githubusercontent.com/assets/1927902/23958451/8b6d9642-0978-11e7-9a73-8930ca1955f1.png)

**Comment by ushenoyuog** [comment]
## Acceptance Criteria

1.  When filtered by category , title should be same as taxonomy term (?)
